sys/include/can/pkt.h

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup    can_dll
+ * @{
+ *
+ *
+ * @file
+ * @brief       CAN memory allocation module
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @author      Toon Stegen <toon.stegen@altran.com>
+ */
+
+#ifndef CAN_PKT_H
+#define CAN_PKT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdatomic.h>
+
+#include "net/gnrc/pktbuf.h"
+
+#include "can/common.h"
+#include "can/can.h"
+#include "msg.h"
+#ifdef MODULE_CAN_MBOX
+#include "mbox.h"
+#endif
+
+typedef struct can_pkt can_pkt_t;
```

Comment:
This line is not needed.
Your can just use :
```c
typedef struct {
    [attributes]
} can_packet_t;
```
(I don't think can_pkt is used elsewhere, didn't check)